app/models/user.rb

```diff
@@ -4,10 +4,10 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: 'author_id'
 
   validates :name, presence: true
-  validates_numericality_of :post_counter, greater_than_or_equal_to: 0
+  validates_numericality_of :posts_count, greater_than_or_equal_to: 0
 
   after_initialize do |user|
-    user.post_counter = 0
+    user.posts_count = 0
```

Comment:
- [ ] Kindly, make sure that you implemented well all your method. `posts_count` should be written as in the database `post_counter`. when I try to add a user, this error is raised:
![image](https://user-images.githubusercontent.com/71874425/193161998-d7135764-e6b1-4ed4-9bd9-c330e61ec674.png)